src/main/java/com/spotify/confidence/ConfidenceValue.java

```diff
@@ -200,6 +276,64 @@ public com.google.protobuf.Value toProto() {
     }
   }
 
+  public static class Timestamp extends ConfidenceValue {
+
+    private final Instant value;
+
+    private Timestamp(Instant value) {
+      this.value = value;
+    }
+
+    @Override
+    public boolean isTimestamp() {
+      return true;
+    }
+
+    @Override
+    public Instant asInstant() {
+      return value;
+    }
+
+    @Override
+    public String toString() {
+      return String.valueOf(value);
+    }
+
+    @Override
+    public com.google.protobuf.Value toProto() {
+      return com.google.protobuf.Value.newBuilder().setStringValue(value.toString()).build();
+    }
+  }
+
+  public static class Date extends ConfidenceValue {
+
+    private final LocalDate value;
+
+    private Date(LocalDate value) {
+      this.value = value;
+    }
+
+    @Override
+    public boolean isDate() {
+      return true;
+    }
+
+    @Override
+    public LocalDate asLocalDate() {
+      return value;
+    }
+
+    @Override
+    public String toString() {
+      return String.valueOf(value);
+    }
+
+    @Override
+    public com.google.protobuf.Value toProto() {
+      return com.google.protobuf.Value.newBuilder().setStringValue(value.toString()).build();
```

Comment:
Do we need to ensure the formatting of this for the backend to parse it correctly?

Reply:
What type of formatting might go wrong?
The docs for LocalDate state: `The output will be in the ISO-8601 format uuuu-MM-dd.`

(We also have serialization tests)